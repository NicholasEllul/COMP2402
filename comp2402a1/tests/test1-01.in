phil
am
i
hi
bob
cat