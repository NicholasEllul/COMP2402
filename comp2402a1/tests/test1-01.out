cat
bob
hi
i
am
phil
